Documentation/networking/overview.md

```diff
@@ -258,9 +258,9 @@ The gateway is configured for the default route, allowing the pod to access exte
 
 ### dhcp
 
-DHCP type requires a daemon to be running on the host.
+The DHCP type requires a special DHCP daemon, part of the [CNI DHCP plugin][cni-dhcp], to be running on the host.
```

Comment:
I think we could avoid a lot of confusion here if we stop calling this a `DHCP daemon`. While the name is technically correct, everybody expects this to mean "the DHCP service which manages all addresses in the subnet", while in our case it is just a middleman service.

Can we perhaps call it a `DHCP (host) agent`?